config.json.example

```diff
@@ -70,6 +70,7 @@
   "reminderScanInterval": 30000,
   "chainMessageCharLimit": "40",
   "forbiddenHighlightChannels": ["0000", "0000"],
+  "tosReminderTimer":"30000",
```

Comment:
This timer specifies how often the timer will catch up on reminders, but there also should be a variable to configure how much time passes before a user gets a reminder after joining. The database is checked every 30 seconds, but users should not get a reminder every 30 seconds - we want to be able to adjust that number. Let's start it at 72 hours for now.

While we're here, please fix the spacing on this line 🙂